test/e2e/e2e.go

```diff
@@ -78,7 +78,7 @@ func RunE2ETests(t *testing.T) {
 	reportDir := framework.TestContext.ReportDir
 	if reportDir != "" {
 		// Create the directory if it doesn't already exists
-		if err := os.MkdirAll(reportDir, 0755); err != nil {
+		if err := os.MkdirAll(reportDir, 0o0755); err != nil {
```

Comment:
If anyone else is wondering about this prefix and needs more than the commit message mention, it's explained here:

https://go.dev/ref/spec#Integer_literals